.travis.yml

```diff
@@ -44,15 +44,15 @@ script:
   - echo "Working copy disk usage"; du -chs .
   - grunt --version
   - grunt help
-  - grunt --quiet --timer
```

Comment:
Was --quiet suppressing errors too? In our case it's used to suppress desktop notifications which are irrelevant from inside travis.